Frontend/Arc-i-Tech/src/app/login/login.component.html

```diff
@@ -1,21 +1,18 @@
-<div class="login-container">
+<div class="box">
```

Comment:
use proper naming conventions for class name eg  login-content

Reply:
Proper naming conventions used